src/qt/walletmodel.cpp

```diff
@@ -519,6 +522,21 @@ bool WalletModel::bumpFee(uint256 hash, uint256& new_hash)
         return false;
     }
 
+    // Short-circuit if we are returning a bumped transaction PSBT to clipboard
+    if (create_psbt) {
+        PartiallySignedTransaction psbtx(mtx);
+        bool complete = false;
+        const TransactionError err = wallet().fillPSBT(psbtx, complete, SIGHASH_ALL, false /* sign */, true /* bip32derivs */);
+        assert(!complete);
+        assert(err == TransactionError::OK);
```

Comment:
Nit: check this first

Reply:
for the future or in case I touch this again: why?

Reply:
The value of complete isn't necessarily meaningful if err is set, I would say, so in that case it's probably less misleading to show a failure message referencing err, where one referencing complete seems potentially very misleading.

Reply:
... actually, for the record, assert()s here kind of scare me -- is that a practice we generally use? Would it be better to do as we do above, and put up a critical messagebox and return failure? Blowing up the process is drastic here; the failure seems well-contained even if surprising.

Reply:
pushed a commit to address the assert concern

Reply:
Also because you changed that in the earlier PR too :-)